Remotion/Core/Tools.UnitTests/Core.Tools.UnitTests.csproj

```diff
@@ -18,7 +18,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACE;NETFRAMEWORK</DefineConstants>
```

Comment:
Revert (since we wen't need conditionals here right now), see https://re-motion.atlassian.net/browse/RMTEAM-301